autogen/oai/client.py

```diff
@@ -472,11 +472,14 @@ def _configure_azure_openai(self, config: Dict[str, Any], openai_config: Dict[st
         openai_config["azure_endpoint"] = openai_config.get("azure_endpoint", openai_config.pop("base_url", None))
 
         # Create a default Azure token provider if requested
-        if openai_config.get("azure_ad_token_provider") == "DEFAULT":
```

Comment:
Default is meant to refer to the `DefaultAzureCredential`, however the parameter that is being changed here is the scope. I feel it should be added as a separate parameter that is correctly named